### ISSUE TITLE
[FEATURE] This is a good app but something is inappropriate. Once I borrow money or lend someone it's treated as income/expense if I made an investment also it's showing same thing.    Please rename the home page income & Expense title into Money in,  Money out.  You need to stablish tow system one is for cash follow where we can see the money in/out during the period and other section where we can see the income/ expenditures during the period.  thank you 

### ISSUE THREAD

**Issue by Habibur3697** [Ivy-Apps/ivy-wallet#3463]
### Please confirm the following:

- [X] I've checked [the current issues](https://github.com/Ivy-Apps/ivy-wallet/issues) for duplicate issues.
- [X] I've requested a single (only one) feature/change in this issue. It complies with the One Request Per GitHub Issue (ORPGI) rule.
- [X] My issue is well-defined and describes how it should be implemented from UI/UX perspective.

### What do you want to be added or improved?

As an user I want to remove the income/ expense title from the home page because this is not cover the definition of income /expenditures this showing total  money inflow & outflow during the period. if O marked as real transaction it is showing all the money in as income which is confusing and illogical.

Say, I get a loan $4000 from my friend during this month and re payment next month then it will overstate my both income and expense.

bottom line:  Please separate income/ expense and money in mony out by adding an option named "Cash follow " 

### Why do you need it?

I am struggling to manage my finance correctly.

### How do you imagine it?

  I am A CMA and well known the accounting compliance so I can't input every money in as an income

**Comment by ivywallet** [comment]
Thank you @Habibur3697 for raising Issue #3463! 🚀
What's next? Read our **[Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md) 📚**.

_Tagging @ILIYANGERMANOV for review & approval 👀_

**Comment by ILIYANGERMANOV** [comment]
Title too long
